locker

```diff
@@ -7,39 +7,64 @@ function lock_image() {
 	echo $NAME@$DIGEST
 }
 
-function lock_dockerfile() {
+function unlock_image() {
+	IMAGE=$1
+	NAME=$(echo $IMAGE | grep '^[^@]\+' -o)
+	echo $NAME
+}
+
+function update_dockerfile() {
+	OPERATION=$1
+	shift
 	FILE=$1
```

Comment:
Don't really need to `shift` here - just use `$1` and `$2`. `shift` is handy for optional arguments.